src/collection-view.js

```diff
@@ -549,14 +561,26 @@ const CollectionView = Backbone.View.extend({
   // This will also update the indices of later views in the collection in order to keep the
   // children in sync with the collection.
   addChildView(view, index) {
+    if (!(view instanceof Backbone.View)) {
```

Comment:
> do we do this kind of bail out elsewhere?

Nope, but if this doesn't happen. The rest of the collection won't render.

I instinctively do these types of checks in my code.

Can can see either case: let it fail loudly or let it slide. Would be nice if we got another vote on leaving or keeping.